components/CategorySelector/index.js

```diff
@@ -0,0 +1,69 @@
+import { useRouter } from "next/router";
+import { useEffect } from "react";
+import styled from "styled-components";
+
+function CategorySelector({ categoryData, wordList, handleFilterWordList }) {
+  const router = useRouter();
+
+  useEffect(() => {
+    handleFilterWordList(wordList);
+  }, [wordList]);
+
+  if (!wordList) {
+    return;
+  }
+
+  function handleDropdownChange(event) {
+    if (event.target.value === "create") {
+      router.push("/categories");
+      return;
+    }
+
+    if (event.target.value === "default") {
+      handleFilterWordList(wordList);
+      return;
+    }
+
+    handleFilterWordList(
+      wordList.filter((entry) => entry.category.includes(event.target.value))
+    );
+  }
+
+  return (
+    <StyledCategoryForm aria-label="choose category">
+      {categoryData && (
+        <StyledCategorySelector
+          onChange={(event) => handleDropdownChange(event)}
+        >
+          <option value="default" defaultValue>
```

Comment:
you either define the defaultValue on the select element: `defaultValue='default'` or you add the `selected` prop to the option.